### ISSUE TITLE
Allow actions to change the resolution field on JIRA issues

### ISSUE THREAD

**Issue by gijsk** [mozilla/jira-bugzilla-integration#200]
Filed per the discussion in #188. We would like to set JIRA tickets to resolution "Done" or "Duplicate" or "Won't do" as appropriate, based on the bugzilla resolution chosen (FIXED/DUPLICATE/WONTFIX).

**Comment by jchaupitre** [comment]
@gijsk, are EPIC in scope of this here? Because for EPIC, in addition to status and resolution can you please also consider updating Epic Status to Done? this will allow EPIC to drop off backlogs and other places in JIRA.

**Comment by jchaupitre** [comment]
FYI, bugs that are being closed by JBI do not have a resolution, and that breaks some of our reporting (e.g. created vs. closed graph), so it would be nice to have this addressed sooner rather than later.

**Comment by leplatrem** [comment]
Since we don't have #77 and #213 yet, we don't have a nice way to integrate this new feature (unless we create a new `default_with_assignee_and_status_and_resolution` action 😵)

So, in order to unblock you, I will just add this step to the `default_with_assignee_and_status` action.
